### DISCUSSION TITLE
极简首页

### DISCUSSION THREAD

**Discussion by hmgdd101** [the1812/Bilibili-Evolved#3024] [提问 / Q&A]
带带，V1的极简首页功能还在comming soon吗，有进展了吗

**Comment by the1812** [comment]
分区里的视频板块做完了, 还剩番剧板块和自定义布局功能

**Reply by hmgdd101** [reply]
刚刚用微信给你表示了一下支持。
收入不高，就当请你喝杯茶，望不嫌弃。

**Reply by the1812** [reply]
感谢支持. 如果用不到分区板块, 可以按照 [v2.1.2 更新公告](https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.2) 的说明提前试用已完成的部分.

**Reply by VergilGao** [reply]
这两天看到更新了分区，设计的很棒
